lib/one_model.ex

```diff
@@ -650,27 +649,51 @@ defmodule OneModel do
   #       the dependency which I tried.
   defp build_fields_list(list, %{include: include}, defaults) when include != [] do
```

Comment:
The assumtion is that include and exclude are mutually exclusive. However it isn't fully validated. Validating this could create issues in existing code so I don't want to change it.